rijkshuisstijl/templates/rijkshuisstijl/components/form/form.html

```diff
@@ -39,6 +39,7 @@
 
                     <fieldset class="form__fieldset{% if fieldset.1.classes %} {{ fieldset.1.classes|join:' ' }}{% endif %}">
                         {% if fieldset.0 %}<legend class="form__legend">{{ fieldset.0 }}</legend>{% endif %}
+                        {% if fieldset.2 %}<p class="form__description">{{ fieldset.2 }}</p>{% endif %}
```

Comment:
Komt dit overeen met de fieldsets definitie zoals in de admin wordt gebruikt? Zo niet, zorgt dit ervoor dat deze 2 niet meer backwards compatible zijn?

Reply:
Op het moment wordt er altijd een derde item verwacht (de omschrijving) in de tuple, ipv normaliter 2 (titel en field_options). Ik denk dat dit nog netter kan door inderdaad de description in de field_options dict te stoppen (net zoals django het doet)

Reply:
Ja, als er al een Django standaard is, eraan houden.